core/src/components/datetime/datetime.tsx

```diff
@@ -1452,14 +1560,14 @@ export class Datetime implements ComponentInterface {
       return [];
     }
 
-    const { workingParts } = this;
+    const { activeParts, workingParts } = this;
 
     return (
       <ion-picker-column-internal
         class="day-column"
         color={this.color}
         items={days}
-        value={workingParts.day || this.todayParts.day}
+        value={(workingParts.day || this.todayParts.day) ?? undefined}
```

Comment:
If you mean the `??` operator, that's this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

`DatetimeParts.day` has a type of `number | null`, while `ion-picker-column-internal`'s `value` prop is `string | number | undefined` and _doesn't_ allow for `null`. This causes a type error since `null` can't be assigned to `value`. The coalescing operator ensures that if `workingParts.day` and `this.todayParts.day` are both `null`, we pass in `undefined` instead, since the coalescing operator returns the right-hand side.

Reply:
Oh right I forgot `day` could be `null`. Ok sounds good.